scripts/build/build.js

```diff
@@ -98,6 +98,14 @@ shell.sed(
   "dist/bin/prettier.js"
 );
 
+shell.echo("Update ISSUE_TEMPLATE.md");
+shell.sed(
+  "-i",
+  /(?!Prettier Version.*?)\d+\.\d+\.\d+/,
```

Comment:
What’s the intention of the negative lookahead (`(?!Prettier Version.*?)`) here? As far as I can tell, it does nothing at all, because a `\d` can never match a `P`.

Reply:
...I mean `(?:Prettier Version)` 😭😭😭, I'll fix it.